.travis.yml

```diff
@@ -319,7 +319,7 @@ install:
   - if [ "$LIBCXX" == "On" ]; then CXX_FLAGS="${CXX_FLAGS} -stdlib=libc++ -I/usr/include/c++/v1/"; fi
   - if [ "$LIBCXX" == "On" ]; then CXX_LINKER_FLAGS="${CXX_FLAGS} -L/usr/lib/ -lc++"; fi
   - cmake .. -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DCMAKE_CXX_FLAGS="${CXX_FLAGS}" -DCMAKE_EXE_LINKER_FLAGS="${CXX_LINKER_FLAGS}" -DRANGES_CXX_STD=$CPP -DRANGE_V3_NO_HEADER_CHECK=1
-  - make VERBOSE=1
+  - travis_wait make VERBOSE=1
```

Comment:
I was trying to fix the travis build as it seems to take a long time on gcc and sometimes there is no output for longer than 20 minutes which is a limit on travis after which the build is killed. 'travis_wait' is printing every few seconds to the output to prevent it. Nevertheless, it's not related to this PR, so sorry about it, just tried to make the PR checks to pass.